apps/user/interfaces/user.interface.ts

```diff
@@ -23,14 +23,35 @@ export interface Organisation {
 }
 
 export interface UsersProfile {
```

Comment:
Let's follow naming convention for interfaces.

Reply:
Resolved